modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,47 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
-----
\ No newline at end of file
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a DualStack config
+
+DualStack can be configured with `ipRanges`.
+
+.Procedure
+
+. Set type to whereabouts
+. Use `ipRanges` to allocate IP addresses.
+
+For example, defining the range as `192.168.10.1/24` excludes the first network address and the last broadcast address.
+
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-shim",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [{
+           "range": "192.168.10.1/24"
+         }, {
+           "range": "abcd: :1/64"
+         }]
+       }
+      }
+
+
+----
+
+. INSERT A CHECK STEP HERE SO WE KNOW WE DID IT CORRECTLY; THIS SHOULD BE A VERIFICATION STEP.
```

Comment:
Example of how we attach this network to a pod: https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/attaching-pod.html#nw-multus-add-pod_attaching-pod

Then, to verify that you have all ip addresses assigned, you can exec:

```
oc exec -it mypod -- ip a
```

Caveat: Required that the `ip` binary is available in the pod.

and also

```
oc describe pod mypod
```

And look for the annotation called `k8s.v1.cni.cncf.io/network-status` (see step 3 in the above docs link) which should show both IP addresses assigned as meta-data associated with the pod.

Reply:
FWIW, and assuming we have `jq` available (not required, but gives a better formatted output) we can:
```bash
oc get pod mypod -ojsonpath="{ @.metadata.annotations.k8s\.v1\.cni\.cncf\.io\/network-status }" | jq
```

This is pretty much what Doug is saying above, but instead of "looking for", you ask for this particular annotation value. 

note: the `| jq` at the end is optional.